clients/python/test/e2e/_utils.py

```diff
@@ -31,7 +32,26 @@ def requires_dev_features(test):
         return pytest.mark.skip(
             (
                 f"{osparc.__version__=}<{str(repo_version)} "
-                "or {osparc_dev_features_enabled()=}"
+                f"or {osparc_dev_features_enabled()=}"
             )
         )(test)
     return test
+
+
+def requires_osparc_version(
```

Comment:
To remind this is adding a skip, I would rename this as `skip_if_osparc_version`

Reply:
OK, makes sense. I will adapt